### DISCUSSION TITLE
Serenity rtl support in versions with bootstrap 5

### DISCUSSION THREAD

**Discussion by zainvegeta** [serenity-is/Serenity#6530] [General]
I am working on a project in need of rtl support.I enabled rtl support with following modifications in layout.cshtml
`@{
    Func<string, HtmlString> json = x => new HtmlString(Serenity.JSON.Stringify(x));
    var hideNav = (string)Context.Request.Query["hideNav"] == "1";
    var themeCookie = Context.Request.Cookies["ProThemeSelection"];
    var theme = themeCookie != null && !themeCookie.IsEmptyOrNull() ? themeCookie : "azure-light";
    var sidebarPaneCollapsed = Context?.Request?.Cookies?["SidebarPaneCollapsed"] == "1" ? " s-sidebar-pane-collapsed" : "";
    var lang = @System.Globalization.CultureInfo.CurrentUICulture.Name;
    var dir = @System.Globalization.CultureInfo.CurrentCulture.TextInfo.IsRightToLeft ? "rtl" : "ltr";

}
<html lang="@lang" dir="@dir" class="pro theme-@theme@(hideNav ? " no-navigation" : "")">`

But seems there are few bugs in rtl mode.I am attaching the screenshots
![image](https://user-images.githubusercontent.com/38037830/189614891-2489227e-800c-4692-8ab7-0912c95d3887.png)

Also sidebar and nav  is not shifting to right side

**Comment by ra-bedo11** [comment]
ماذا فعلت فى حل هذة المشكله ؟

**Comment by ra-bedo11** [comment]
عندى مشكله فى سيرينتى فى كور 6
لما اجى اضيف اللغه العربية ar            ar-EG
للاسف لما باجى بقى اختار عشان اقلب الموقع عربى مش بيرضى 
يقلب معاى الصفحة من الشمال لليمين
rtl ??

**Comment by zainvegeta** [comment]
??????

**Reply by ra-bedo11** [reply]
Please 
I want the fields in the Arabic form to be designed like the English How ?

**Comment by VictorTomaili** [comment]
Please use English in conversation.

**Reply by ra-bedo11** [reply]
Please 
I want the fields in the Arabic form to be designed like the English How ?
